src/features/events_list/components/EventsPanel/EventsPanel.tsx

```diff
@@ -0,0 +1,85 @@
+import { Panel, PanelIcon } from '@konturio/ui-kit';
+import { useCallback } from 'react';
+import clsx from 'clsx';
+import { Disasters24 } from '@konturio/default-icons';
+import { useAtom } from '@reatom/react';
+import { i18n } from '~core/localization';
+import { userResourceAtom } from '~core/auth';
+import { AppFeature } from '~core/auth/types';
+import { IS_MOBILE_QUERY, useMediaQuery } from '~utils/hooks/useMediaQuery';
+import { useAutoCollapsePanel } from '~utils/hooks/useAutoCollapsePanel';
+import { panelClasses } from '~components/Panel';
+import { useHeightResizer } from '~utils/hooks/useResizer';
+import { useShortPanelState } from '~utils/hooks/useShortPanelState';
+import { MIN_HEIGHT } from '../../constants';
+import { FullState } from '../FullState/FullState';
+import { ShortState } from '../ShortState/ShortState';
+import s from './EventsPanel.module.css';
+
+export function EventsPanel({
+  currentEventId,
+  onCurrentChange,
+}: {
+  currentEventId?: string | null;
+  onCurrentChange: (id: string) => void;
+}) {
+  const { panelState, panelControls, setPanelState } = useShortPanelState();
+
+  const isOpen = panelState !== 'closed';
+  const isMobile = useMediaQuery(IS_MOBILE_QUERY);
+  const [{ data: userModel }] = useAtom(userResourceAtom);
+  const handleRefChange = useHeightResizer(
+    (isOpen) => !isOpen && setPanelState('closed'),
+    isOpen,
+    MIN_HEIGHT,
+  );
+  const openFullState = () => setPanelState('full');
```

Comment:
openFullState and onPanelIconClick do the same thing 
feels like 1 of them is redundant